### PR TITLE
updating the dev-qa codes, and deleting unused code.

### PR DESCRIPTION
**Pull request by vaslee**
updating the dev-qa codes, and deleting unused code.

### CODE REVIEW

File: StellarApp/Protocol/OrbitRotationFiles.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
+//
+//  OrbitRotationFiles.swift
```
Comment (vhart) [inline StellarApp/Protocol/OrbitRotationFiles.swift]: Conventionally we don't say `File` in a file name unless it describes a `File` object

File: StellarApp/Protocol/OrbitRotationFiles.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+struct RotationVector {
+    let x: CGFloat, y: CGFloat, z: CGFloat
+}
+
+protocol Rotatable {
+    func rotate()
+    func stopRotating()
+}
+
+class RotatingNode: SCNNode, Rotatable {
```
Comment (vhart) [inline StellarApp/Protocol/OrbitRotationFiles.swift]: if we're not using this stuff you can go ahead and delete it. If we need it later on it'll be in the git history

**Review by vhart — APPROVED** [review]
A few other comments were added to your other PR. But for right now this LGTM

File: StellarApp/Protocol/OrbitRotationFiles.swift (hunk 2 above)
Comment (vaslee) [reply StellarApp/Protocol/OrbitRotationFiles.swift]: ok

Comment (vaslee) [reply StellarApp/Protocol/OrbitRotationFiles.swift]: I want to keep it for now till we talk about tomorrow.

**Pull request merged**